urdf/ur.ros2_control.xacro

```diff
@@ -30,6 +30,8 @@
         </xacro:if>
         <xacro:if value="${use_fake_hardware}">
           <plugin>isaac_ros2_control/IsaacSystem</plugin>
+          <param name="joint_commands_topic">/isaac_joint_commands</param>
+          <param name="joint_states_topic">/isaac_joint_states</param>
           <param name="fake_sensor_commands">${fake_sensor_commands}</param>
           <param name="state_following_offset">0.0</param>
```

Comment:
Do we need these parameters @JafarAbdi?

Reply:
No, we don't need those

Reply:
Should I clean them up or wait until we spend the time to add a `sim_isaac` variable?